src/usb_moded-worker.c

```diff
@@ -639,34 +636,35 @@ worker_set_requested_mode_locked(const char *mode)
     LOG_REGISTER_CONTEXT;
 
     bool changed = false;
-    const char *prev = worker_get_requested_mode_locked();
-
-    if( !g_strcmp0(prev, mode) )
-        goto EXIT;
 
-    log_debug("requested_mode: %s -> %s", prev, mode);
-    g_free(worker_requested_mode),
-        worker_requested_mode =  g_strdup(mode);
-    changed = true;
+    if( g_strcmp0(worker_requested_mode, mode) ) {
+        log_debug("requested_mode: %s -> %s", worker_requested_mode, mode);
+        g_free(worker_requested_mode),
+            worker_requested_mode =  g_strdup(mode);
```

Comment:
Existing thing and used elsewhere, but I'm not sure I really like this use of comma operator instead of just doing things in separate statements. 

Also extra space here after assignment.

Reply:
Space removed. Comma stays.